origocli/commands/datasets.py

```diff
@@ -86,7 +88,7 @@ def dataset(self):
                 list["dataset"] = set
                 list["versions"] = versions
                 list["latest"] = latest
-                self.print("", list)
+                self.print("", json.dumps(list))
```

Comment:
We should probably have a top-level flag to turn off/on pretty printing with colors, whistles and bells and whatnot.

If we go for using `argparse` instead of our own argument parsing it would be trivial to add I think.